composer.json

```diff
@@ -131,31 +63,25 @@
     "extra": {
         "symfony-app-dir": "src/app",
         "symfony-web-dir": "src/web",
+        "symfony-assets-install": "relative",
         "installer-paths": {
             "src/modules/{$vendor}/{$name}": ["type:zikula-module"],
             "src/themes/{$vendor}/{$name}": ["type:zikula-theme"]
-        }
-    },
-    "repositories": [
-        {
-            "type": "vcs",
-            "url": "https://github.com/zikula-modules/Legal"
-        },
-        {
-            "type": "vcs",
-            "url": "https://github.com/zikula-modules/Profile"
         },
-        {
-            "type": "vcs",
-            "url": "https://github.com/FrDH/jQuery.mmenu"
-        },
-        {
-            "type": "vcs",
-            "url": "https://github.com/aFarkas/webshim"
-        },
-        {
-            "type": "vcs",
-            "url": "https://github.com/corejavascript/typeahead.js"
+        "merge-plugin": {
+            "include": [
+                "src/lib/Zikula/Bundle/*/composer.json",
+                "src/system/*/composer.json",
+                "src/modules/*/composer.json",
+                "src/modules/*/*/composer.json",
+                "src/themes/*/composer.json",
+                "src/themes/*/*/composer.json"
```

Comment:
that could be a problem because there is no guarantee of the composer.json placement level